docs/bundles/SyliusResourceBundle/forms.rst

```diff
@@ -43,7 +53,8 @@ Create the form's service :
             - { name: form.type }
         arguments: ['%app.model.book.class%']
```

Comment:
We should mention about optional argument with validation groups. 

Also a note, that the service definition is only needed when one has extended `AbstractResourceType` or has some custom dependencies.